docs/reference/env_variables.rst

```diff
@@ -0,0 +1,177 @@
+.. meta::
+    :description: HIP environment variables reference
+    :keywords: AMD, HIP, environment variables, environment, reference
+
+*************************************************************
+HIP environment variables
+*************************************************************
+
+In this section, the reader can find all the important HIP environment variables. The full collection of the ROCm environment variables is on the :doc:`ROCm environment variables page<rocm:reference/env-variables>`.
```

Comment:
I think this could use a little clarification as to the environment variables displayed here versus the additional variables shown on the referenced pages. Why are these environment variables worth looking at here, but not the others? 

Also, I think the reference needs a space: 
:doc:`ROCm environment variables page <rocm:reference/env-variables>`